[Feature][Format] Improve maxwell_json,canal_json,debezium_json format support merge update_before and update_after (#9805)
apache/seatunnel: docs/en/connector-v2/sink/LocalFile.md

| encoding                              | string  | no       | "UTF-8"                                    | Only used when file_format_type is json,text,csv,xml.                                                                                                                  |
| schema_save_mode                      | string  | no       | CREATE_SCHEMA_WHEN_NOT_EXIST               | Existing dir processing method                                                                                                                                         |
| data_save_mode                        | string  | no       | APPEND_DATA                                | Existing data processing method                                                                                                                                        |

> | merge_update_event                    | boolean | no       | false                                      | Only used when file_format_type is canal_json,debezium_json,maxwell_json.set true,when serialize data,UPDATE_AFTER and UPDATE_BEFORE event will merge into UPDATE data |

Reviewer: ```suggestion
| merge_update_event                    | boolean | no       | false                                      | Only used when file_format_type is canal_json,debezium_json or maxwell_json. When value is true, the UPDATE_AFTER and UPDATE_BEFORE event will be merged into UPDATE event data |
```
